docs/docs/user-guide/contributing/Writing Documentation/jupyter-notebooks.ipynb

```diff
@@ -127,16 +127,19 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2.3.0a0+ebedce2\n"
+      "[0 7 6 4 4]\n"
      ]
     }
    ],
    "source": [
     "#NBVAL_CHECK_OUTPUT\n",
     "# pragma: allowlist secret\n",
     "\n",
-    "import torch\n",
-    "print(torch.__version__)"
+    "import numpy as np\n",
+    "\n",
+    "\n",
+    "rng = np.random.default_rng(42)\n",
+    "print(rng.integers(0, 10, size=5))"
```

Comment:
this is also likely to change with time/hardware/etc. Maybe `print(np.arange(5))` would be safer?

Reply:
in fact you could probably yolo edit the json yourself in this diff without re-running if you did that, since it should follow the same format as a list of integers.

Reply:
Sure; numpy has made some guarantees around random reproducibility but i agree, better not to tempt fate